WordPress/wordcamp.org
# Update text domain in CampTix to wordcamporg

Now that CampTix is part of the WordCamp codebase, it should share the same text domain as the rest of the code.

This makes two changes:

* Changes the text domain in every usage of a translation function from `camptix` to `wordcamporg`
* Removes the method that loads a text domain from the CampTix plugin, since that is taken care of in mu-plugins

**To test**

It would be difficult and very time consuming to try and make sure every translatable string is correctly getting translated with its new text domain. I don't think we should attempt that here. However, there were some false positives in the code when doing a search and replace for the text domain. We should make sure none of those false positives got replaced. So:

* The handle for the main JS and CSS files for the plugin is the same as the old text domain. So make sure scripts and styles are loading and working correctly. A camp's Attendees page is a good place to do this, because it loads both **camptix.js** and **camptix.css**.
* There is a global instance of the `CampTix_Plugin` class that is assigned via `$GLOBALS['camptix']`, and the variable is also occasionally referenced this way. So click around the CampTix UI and make sure there aren't any fatal errors from missing variables.
